### PR TITLE
add security

### PR DESCRIPTION
**Pull request by alexander-schranz**
**Tasks:**
- [x] done
- [x] got feedback

**Informations:**

| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| Tests pass? | none |
| Fixed tickets | none |
| Doc | none |

### CODE REVIEW

File: Controller/OrderController.php [hunk 1]
```diff
@@ -297,4 +298,12 @@ private function getStatusFieldDescriptor()
...
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function getSecurityContext()
+    {
+        return 'sulu.sales_order.orders';
+    }
+
```
Comment (danrot) [inline Controller/OrderController.php]: no new line here

**Comment by danrot** [comment]
Have the tests already been failing before?

**Comment by alexander-schranz** [comment]
The tests failed before. in #21 I did updated the tests from DatabaseTestCase to SuluTestCase but some  tests are still failing there.

**Comment by danrot** [comment]
Ok, same as for https://github.com/sulu-cmf/SuluProductBundle/pull/74, please merge it yourself i coordination with @turbo-ele

**Pull request merged**